src/flow_solver/flow_solver_cases/periodic_turbulence.cpp

```diff
@@ -734,53 +743,82 @@ void PeriodicTurbulence<dim, nstate>::compute_unsteady_data_and_write_to_table(
         const std::shared_ptr <dealii::TableHandler> unsteady_data_table)
 {
     // Compute and update integrated quantities
-    this->compute_and_update_integrated_quantities(*dg);
-    // Get computed quantities
+    if (do_calculate_overintegrated_quantities) this->compute_and_update_integrated_quantities(*dg);
```

Comment:
maybe rename to `do_compute_and_update_integrated_quantities`; but this could cause issues if its false and an CFL number is used for the time step since the max wavelength is updated in the function `compute_and_update_integrated_quantities(*dg);`

Reply:
That's definitely an oversight on my part, I forgot that the maximum wave speed is updated in that loop.
Per discussion, I have removed this parameter.